norduniclient/core.py

```diff
@@ -165,6 +165,10 @@ def query_to_iterator(manager, query, **kwargs):
             yield d
 
 
+def _node_to_dict(node):
```

Comment:
Can we call it neo4j_entity_to_dict (or perhaps _entity_to_dict) as it is used for relationships also?